e2e/pipelines/test_gradient_rag_pipelines.py

```diff
@@ -0,0 +1,92 @@
+import os
+import json
+import pytest
+
+from haystack.preview import Pipeline, Document
+from haystack.components.embedders.gradient_document_embedder import GradientDocumentEmbedder
+from haystack.components.embedders.gradient_text_embedder import GradientTextEmbedder
+from haystack.preview.document_stores import InMemoryDocumentStore
+from haystack.preview.components.writers import DocumentWriter
+from haystack.preview.components.retrievers import InMemoryEmbeddingRetriever
+from haystack.preview.components.generators.gradient.base import GradientGenerator
+from haystack.components.builders.answer_builder import AnswerBuilder
+from haystack.components.builders.prompt_builder import PromptBuilder
```

Comment:
The `preview` package is gone, here is the fix:

```suggestion
from haystack import Pipeline, Document
from haystack.components.embedders.gradient_document_embedder import GradientDocumentEmbedder
from haystack.components.embedders.gradient_text_embedder import GradientTextEmbedder
from haystack.document_stores import InMemoryDocumentStore
from haystack.components.writers import DocumentWriter
from haystack.components.retrievers import InMemoryEmbeddingRetriever
from haystack.components.generators.gradient.base import GradientGenerator
from haystack.components.builders.answer_builder import AnswerBuilder
from haystack.components.builders.prompt_builder import PromptBuilder
```

Reply:
looks like the GradientGenerator wasn't moved out of the preview anyways, so I moved it. All the Gradient tests are passing now in my box!